Gopkg.toml

```diff
@@ -61,6 +61,14 @@ ignored = [
   name = "github.com/golang/glog"
   source = "github.com/istio/glog"
 
+[[override]]
```

Comment:
does constraint not work here?  https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md#override

Reply:
Override makes sure that even if it does not find imports in the go code, it would still add this package to vendor. This tool is not part of your build system so when you run dep ensure it does not look at imports done by this code. And since the rest of istio does not import helm it has to be forced to add this package.